Tools/buildstream/elements/freedesktop-sdk.bst

```diff
@@ -3,7 +3,7 @@ sources:
 - kind: git_tag
   url: gitlab_com:freedesktop-sdk/freedesktop-sdk.git
   track: 'release/22.08'
-  ref: freedesktop-sdk-22.08.2.1-168-g9b732aab03152bc450e206c630d9dfd187c318c3
+  ref: freedesktop-sdk-22.08.3-64-g754b0e0c6dc0e648a0f09fb5773612339555352f
```

Comment:
Out of curiosity, how do you keep track of the changes? Inspect manually what happened between tags? Rely on bst output? Just wondering about your specific workflow :-)

Reply:
i run `bst-wrapper track freedesktop-sdk.bst` and it updates the ref base on what's being tracked.

Reply:
I also check their git log and NEWS file but I don't do it exhaustively for minor releases, I trust them.